src/terminal/sys/unix.rs

```diff
@@ -53,6 +53,10 @@ pub(crate) fn window_size() -> io::Result<WindowSize> {
     };
 
     if wrap_with_result(unsafe { ioctl(fd, TIOCGWINSZ.into(), &mut size) }).is_ok() {
+        if size.ws_row == 0 && size.ws_col == 0 {
+            size.ws_row = env::var("LINES").unwrap_or("0".to_string()).parse::<u16>().unwrap();
+            size.ws_col = env::var("COLUMNS").unwrap_or("0".to_string()).parse::<u16>().unwrap();
```

Comment:
```suggestion
            size.ws_row = env::var("LINES").map_or(Ok(0), |v| v.parse()).unwrap_or(0);
            size.ws_col = env::var("COLUMNS").map_or(Ok(0), |v| v.parse()).unwrap_or(0);
```

Reply:
Thanks for the suggestion, I changed the code accordingly.